sale_quotation_number/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     "name": "Sale Quotation Numeration",
     "summary": "Different sequence for sale quotations",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.0",
```

Comment:
@dsolanki-initos Could you put this change in migration commit ? Thanks

Reply:
Sure, @rousseldenis